### ISSUE TITLE
Use cases feedback - find biosamples by phenotype/disease

### ISSUE THREAD

**Issue by mcourtot** [Ensembl/schemas#2]
I would rephrase "Find all biosamples for an individual with an annotation of polydactyly (phenotype)" to "Find all biosamples for an individual with a specific phenotype or disease" to be a little bit broader. We should also keep in mind that usually disease are only at the individual level, while phenotypes can be both individual and sample.

**Comment by mcourtot** [comment]
Maybe also consider positive/negative queries, e.g. all individuals that have X and NOT Y

**Comment by kozbo** [comment]
Logical operations move us in the direction of implementing an SQL query, which we will never do as a direct SQL portal. Is there a level of implementation that provides enough functionality without taking on all the complexity of a full featured logical query language?

**Comment by mcourtot** [comment]
Phenopackets support [negation](https://github.com/phenopackets/phenopacket-format/blob/master/examples/level-1/journal-example-l1.json#L207) - so if we were able to export from GA4GH to PXF we should be able to support that type of query. Other option is to have the negation as part of the characteristics objects following a similar model, i.e. prefixing ontologTerm(s) by "not" in the array.

**Comment by lairdm** [comment]
I agree with Melanie, negation doesn't necessarily mean exposing the underlying implementation. I can see allowing negative server side filtering on result sets.
